ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/RDXZEDSVORecorderPanel.java

```diff
@@ -30,23 +31,28 @@ public class RDXZEDSVORecorderPanel
    public RDXZEDSVORecorderPanel(ROS2Helper ros2Helper)
    {
       this.ros2Helper = ros2Helper;
+      
       ros2Helper.subscribeViaCallback(PerceptionAPI.ZED_SVO_CURRENT_FILE, message -> this.latestMessage = message);
    }
 
-   public void update()
+   public void createWithOverlayPanel(RDX3DPanel panel3D)
    {
-      RDXBaseUI baseUI = RDXBaseUI.getInstance();
+      panel3D.addOverlayPanel(PANEL_NAME, this::renderImGuiWidgets);
+   }
 
-      boolean overlayPanelExists = baseUI.getPrimary3DPanel().overlayPanelExists(PANEL_NAME);
+   public void createWithRegularPanel(RDXBaseUI baseUI)
```

Comment:
Added option to create this as a regular panel.

Reply:
I would like to revert this because I would argue SVO recorder should always be an overlay

Reply:
I find it valueable to be a regular panel in the scene graph demo. When it's collapsed you can't see all the widgets and I really find that useful and also would like to add more features, like stepping through 1 frame at a time. Currently the detection manager and scene graph are hard to debug and I'd like to make it SCS-like where you can step through the data frame by frame and inspect state.

Reply:
I think it makes the user experience less consistent and annoying to use